src/intervals/functions.jl

```diff
@@ -159,26 +159,44 @@ function exp(a::Interval{Float64})
     Interval(exp(a.lo, RoundDown), exp(a.hi, RoundUp))
 end
 
-function exp(a::Interval{BigFloat})
-    isempty(a) && return a
-    @controlled_round(BigFloat, exp(a.lo), exp(a.hi))
+for fn in (:exp2, :exp10)
+    @eval begin
+        function ($fn)(a::Interval{Float64})
+            isempty(a) && return a
+            res = with_bigfloat_precision(53) do
```

Comment:
I think we should introduce a `big53` function.
Then you can write something like the following to define this kind of function.

```julia
exp(a::Interval{Float64}) = float(exp(big53(a)))
```